framework/src/mfem/utils/PropertyManager.C

```diff
@@ -0,0 +1,134 @@
+#ifdef MFEM_ENABLED
+
+#include "PropertyManager.h"
+#include "MooseError.h"
+#include <algorithm>
+
+namespace Moose::MFEM
```

Comment:
This class is an example where it would be important to have the context of the requesting moose object. Ideally - in the form of an interface. That way, when we hit errors in receiving properties, the error gets reported with the context of the calling object instead of as a general error.

Reply:
Opened https://github.com/idaholab/moose/issues/30327